packages/block-library/src/navigation-link/edit.js

```diff
@@ -297,12 +294,8 @@ export default function NavigationLinkEdit( {
 	const blockProps = useBlockProps( {
 		ref: listItemRef,
 		className: classnames( {
-			'is-editing':
-				( isSelected || isParentOfSelectedBlock ) &&
-				// Don't show the element as editing while dragging.
-				! isDraggingBlocks,
-			// Don't select the element while dragging.
-			'is-selected': isSelected && ! isDraggingBlocks,
+			'is-editing': isSelected || isParentOfSelectedBlock,
+			'is-selected': isSelected,
```

Comment:
Why do we assign the `is-selected` class when `useBlockProps` is responsible for this?

And could we also revert the change here: https://github.com/WordPress/gutenberg/commit/748c632feccbffb51a6e3c79972a447b71211a0c#diff-4501da117949f251043d70ffccf02c54af2a99fb80ccd33eb2d40175acdfc833L153?

Reply:
I don't know to be honest. Might just have been a visual thing.

It was originally added in this PR - https://github.com/WordPress/gutenberg/pull/17832 - which has a lot of comments, it might have been discussed in there at some point.

I'll remove that line.

Reply:
Ah, sorry, should have linked this one too: https://github.com/WordPress/gutenberg/commit/748c632feccbffb51a6e3c79972a447b71211a0c#diff-4501da117949f251043d70ffccf02c54af2a99fb80ccd33eb2d40175acdfc833R165

Reply:
Have removed both of the modified lines, and it still seems to work, so hopefully there's no drama from that change (cd3aa0c).

Reply:
If there's a problem we look for a pure CSS solution. :)